armi/operators/operator.py

```diff
@@ -650,6 +654,10 @@ def interactAllCoupled(self, coupledIteration):
         This is distinct from loose coupling, which would simply uses the temperature values from the previous timestep
         in the current flux solution. It's also distinct from full coupling where all fields are solved simultaneously.
         ARMI supports tight and loose coupling.
+
+        .. impl:: Physics coupling is driven from Operator.
+            :id: I_ARMI_OPERATOR_PHYSICS2
```

Comment:
Same as above comment, is this supposed to be `1`?

Reply:
Fixed!